modules/bot/src/main/kotlin/dev/proxyfox/bot/command/MemberCommands.kt

```diff
@@ -50,8 +55,15 @@ object MemberCommands {
         createGlobalChatInputCommand("member", "Manage or create a system member!") {
             subCommand("create", "Create a member") {
                 name()
+                system()
                 runs {
-                    val system = database.fetchSystemFromUser(getUser())
+                    val id = value.interaction.command.strings["system"]
+                    val system =
+                        if (id == null)
+                            database.fetchSystemFromUser(getUser())
+                        else
+                            database.fetchSystemFromId(id)
+                                ?: database.fetchSystemFromUser(id.toULongOrNull() ?: 0UL)
```

Comment:
Return instead of trying to fetch ID of 0

This seems to also be boilerplate; possible to reduce the amount of copy-pasting.